capstone-app/src/app/modules/history/wiki-media/wiki-media.component.scss

```diff
@@ -1,35 +1,57 @@
-.history {
-  background-color: #f5f5f5;
-  margin: 15px;
+
+.history-title.mat-card-title {
+  color: black;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 45px;
+  font-weight: 590;
+  margin-left: 3%;
+  margin-right: 3%;
+  padding-bottom: 30px;
+  text-align: center;
 }
 
-.history-title {
-  font-style: italic;
-  margin: 15px;
+.history-body.mat-card {
+  background-color: white;
+  color: black;
+  font-family: 'Roboto', sans-serif;
+  font-size: 21px;
+  height: auto;
+  line-height: 30px;
+  margin-bottom: 2%;
```

Comment:
I'd recommend using a pixel amount for all margins instead of a percent. A minor css change in the future could throw off the layout of the site with a percent margin.

Reply:
Here and in all other files